lib/core/src/lib/datatable/components/json-cell/json-cell.component.spec.ts

```diff
@@ -15,15 +15,12 @@
  * limitations under the License.
  */
 
-import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { ObjectDataTableAdapter } from '../../data/object-datatable-adapter';
-import { ObjectDataColumn } from '../../data/object-datacolumn.model';
-import { CoreTestingModule } from '../../../testing/core.testing.module';
-import { JsonCellComponent } from './json-cell.component';
-import { TranslateModule } from '@ngx-translate/core';
-import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { CoreTestingModule, JsonCellComponent, ObjectDataColumn, ObjectDataTableAdapter } from '@alfresco/adf-core';
```

Comment:
this type of import is incorrect as might produce circular loops

Reply:
done